DevEdu.API/DevEdu.Business.Tests/Data/LessonData.cs

```diff
@@ -0,0 +1,142 @@
+                Teacher = new UserDto
+                {
+                    Id = 3
+                },
+                LinkToRecord = "http://link.com"
+            };
+        }
+
+        public static LessonDto GetUpdatedLessonDto()
+        {
+            return new LessonDto
+            {
+                Id = LessonId,
+                Date = DateTime.ParseExact("06.07.2021", _dateFormat, CultureInfo.InvariantCulture),
+                TeacherComment = "Good",
+                LinkToRecord = "http://link.com"
+            };
+        }
+
+        public static LessonDto GetSelectedLessonDto()  
+        {
+            return new LessonDto
+            {
+                Id = LessonId,
+                Date = DateTime.ParseExact("06.07.2021", _dateFormat, CultureInfo.InvariantCulture),
+                TeacherComment = "Good",
+                Teacher = new UserDto
+                {
+                    Id = 3
+                },
+                LinkToRecord = "http://link.com"
+            };
+        }
+
+        public static List<LessonDto> GetLessons() 
+        {
+            return new List<LessonDto>{
+                new LessonDto
+                {
+                    Id = 2,
+                    Date = DateTime.ParseExact("06.07.2021", _dateFormat, CultureInfo.InvariantCulture),
+                    TeacherComment = "Good",
+                    Teacher = new UserDto {
+                        Id = 3,
+                        FirstName = "Olga",
+                        LastName = "Ivanovna",
+                        Email = "olga@mail.ru",
+                        Photo = " http://photo.jpg"
+                    },
+                    Topics = new List<TopicDto>()
+                    {
+                        new TopicDto{
+                            Id = 4,
+                            Name = "oop"
+                        }
+                    }
+                },
+                new LessonDto
+                {
+                    Id = 5,
+                    Date = DateTime.ParseExact("12.07.2021", _dateFormat, CultureInfo.InvariantCulture),
+                    TeacherComment = "Good",
+                    Teacher = new UserDto {
+                        Id = 3,
+                        FirstName = "Olga",
+                        LastName = "Ivanovna",
+                        Email = "olga@mail.ru",
+                        Photo = " http://photo.jpg"
+                    },
+                    Topics = new List<TopicDto>()
+                    {
+                        new TopicDto{
+                            Id = 6,
+                            Name = "service"
+                        }, 
+                        new TopicDto{
+                            Id = 7,
+                            Name = "tests"
+                        }
+                    }
+                }
+            };
+        }
+
+        public static List<StudentLessonDto> GetStudents() 
```

Comment:
the name of this method is confusing
you are not returning students here, it's a bit more like attendance-list